bump protobuf version

darshkpatel opened the pull request:
Bump protobuf version so users don't face issues if they have an older version already installed on their systems.
Ref: #2234, #2212, #1848
Ref: [stackoverflow](https://stackoverflow.com/questions/61922334/how-to-solve-attributeerror-module-google-protobuf-descriptor-has-no-attribu)
`pip install --upgrade protobuf`


 Screenshot for reference:
<img width="1439" alt="Screenshot 2020-10-24 at 12 59 50 PM" src="https://user-images.githubusercontent.com/11258286/97071002-8e662c00-15fa-11eb-94c2-381e7556c361.png">


**Contribution License Agreement**

By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

darshkpatel committed:
bump protobuf version

Bump protobuf version so users don't face issues if they have an older version already installed on their systems.
Ref: #1764 
Ref: [stackoverflow](https://stackoverflow.com/questions/61922334/how-to-solve-attributeerror-module-google-protobuf-descriptor-has-no-attribu)
`pip install --upgrade protobuf`

akrolsmir commented on lib/Pipfile:
```diff
@@ -41,7 +41,7 @@ numpy = "*"
 pandas = ">=0.21.0"
 pillow = ">=6.2.0"
-protobuf = ">=3.6.0"
+protobuf = ">=3.13.0"
```
Thanks for submitting this change, @darshkpatel! Upon discussion with the team, we're hesitant to require such a huge jump in protobuf version; it could break other dependencies (your screenshot shows mitmproxy not expecting 3.13.0 haha)

Could we instead just forbid the bad version of protobuf (3.11, I believe)? And add a comment so we know why:

```
# protobuf version 3.11 is incompatible, see https://github.com/streamlit/streamlit/issues/2234
protobuf = ">=3.6.0, !=3.11"
```

akrolsmir reviewed (changes requested):
Thanks for helping improve Streamlit! Just one note above.

darshkpatel committed:
forbid the bad version

as mentioned in https://github.com/streamlit/streamlit/pull/2264#discussion_r513678216

darshkpatel commented:
@akrolsmir Pushed : D

akrolsmir reviewed (approved):
Awesome, thanks!
